docs/src/content/docs/challenges/angular/50-bug-effect-signal.md

```diff
@@ -13,10 +14,27 @@ sidebar:
 
 ## Information
 
-In this basic exercise, we aim to display an alert whenever at least one checkbox is checked.
+In this basic exercise, we aim to display an alert whenever at least one checkbox is checked. You are in the process of buying a MacBook, which can be upgraded with some extras, like more drive space, more RAM or a better GPU.
+
+<img width="889" alt="Bildschirmfoto 2024-05-09 um 08 57 57" src="https://github.com/svenson95/angular-challenges/assets/46655156/d78f42a5-9064-4a33-bb8c-c0433bd6966d">
 
 ## Statement
 
-The alert correctly triggers when clicking on each checkbox separately. However, if you first click on one checkbox and then click on a second one, the alert fails to appear. Why does this happen?
+The actual implementation doesn't work as expected, your task is to fix the issue. Your team exposed a bug, there is a alert to be shown if atleast one of the three checkboxes is checked. But if the first one is checked, the other two checkboxes gets checked without displaying the alert. Why does this happen?
+
+The objective of this challenge is to understand the issue and fix the problem, preventing the alert from appearing when the second checkbox is clicked.
+
+## Acceptance Criteria
+
+To ensure this feature works properly, try this out to reproduce the bug after solving the challenge, to check if the bug is gone.
+
+- Check box 1 (Alert should be shown)
+- Check box 2 (Alert should be shown)
+- Uncheck box 1
+- Check box 3 (Alert should be shown)
+- Uncheck box 2
+- Uncheck box 3
```

Comment:
If we do it this way. the only way to solve the challenge is by having multiple effects. Otherwise even if we uncheck one checkbox, the alert will be shown.

Reply:
That would be okay for you? if so, we have to update our solutions

Reply:
It's just that the challenge was just about the issue on the effect. But this doesn't really matter. We can keep it this way. I just don't know how to do it simply with one effect

Reply:
Yeah you're right. I suggest to change line 40 to the following. So we can find different ways to solve this, maybe with one effect, i'll try this.

> Try to implement this feature with a `computed` signal.